.buildkite/pipeline-raw.yml

```diff
@@ -154,6 +155,17 @@ steps:
      queue: "docker"
    only_changes: *native_packaging_changes_regexes
 
+ - label: test gen_systemd_service_file.py script
+   commands:
+   - eval "$SET_VERSION"
+   - ./gen_systemd_service_file.py tezos-node
```

Comment:
```suggestion
   - nix run -f . pkgs.python3 -c ./gen_systemd_service_file.py tezos-node
```

Reply:
ah, forgot about that, done :ok_hand:

Reply:
Ugh, that's actually more complicated, docker and non-docker BK agents have different nix versions :confused: 
One wants `nix run`, the other `nix shell`